lib/ansible/module_utils/ec2.py

```diff
@@ -255,7 +255,11 @@ def boto_fix_security_token_in_profile(conn, profile_name):
 
 
 def connect_to_aws(aws_module, region, **params):
-    conn = aws_module.connect_to_region(region, **params)
+    try:
+        conn = aws_module.connect_to_region(region, **params)
+    except boto.provider.ProfileNotFoundError:
+        raise AnsibleAWSError(
+             "AWS Profile {0} not found.  Please fix and retry.".format(params["profie_name]))
```

Comment:
profile_name is mispelled (and there is a missing quote after profile_name as the tests suggest).

No need for two spaces after the full stop.